source/guides/assets/content-delivery-network.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Lotus - Guides - Assets Content Delivery Network (CDN)
+---
+
+# Assets
+
+## Content Delivery Network (CDN)
+
+A Lotus application can serve assets from a [Content Delivery Network](https://en.wikipedia.org/wiki/Content_delivery_network) (CDN).
+This feature is useful in _production_ environment, where we want to speed up static assets serving.
+
+In order to take advantage of this feature, we need to specify CDN settings.
```

Comment:
can we add a notice here that users should take care of the uploading assets to CDN prior to configuring CDN?